From f6fa9985900ee757a2cfce0f472ff1fc84463242 Mon Sep 17 00:00:00 2001
From: Yi Kong <yikong@google.com>
Date: Tue, 5 Jul 2022 17:26:34 +0800
Subject: [PATCH] Fix tests with non-default CLANG_DEFAULT_LINKER

Force -fuse-ld option, as some other tests in the same file do.
---
 clang/test/Driver/instrprof-ld.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/clang/test/Driver/instrprof-ld.c b/clang/test/Driver/instrprof-ld.c
index 3a6cdab10c3..9a58cd3a0be 100644
--- a/clang/test/Driver/instrprof-ld.c
+++ b/clang/test/Driver/instrprof-ld.c
@@ -154,7 +154,7 @@
 //
 // RUN: %clang -### %s 2>&1 \
 // RUN:     --target=i386-pc-win32 -fprofile-instr-generate \
-// RUN:     -resource-dir=%S/Inputs/resource_dir \
+// RUN:     -resource-dir=%S/Inputs/resource_dir -fuse-ld=link \
 // RUN:   | FileCheck --check-prefix=CHECK-WINDOWS-I386 %s
 //
 // CHECK-WINDOWS-I386: "{{.*}}link{{(.exe)?}}"
@@ -162,7 +162,7 @@
 //
 // RUN: %clang -### %s 2>&1 \
 // RUN:     --target=x86_64-pc-win32 -fprofile-instr-generate \
-// RUN:     -resource-dir=%S/Inputs/resource_dir \
+// RUN:     -resource-dir=%S/Inputs/resource_dir -fuse-ld=link \
 // RUN:   | FileCheck --check-prefix=CHECK-WINDOWS-X86-64 %s
 //
 // CHECK-WINDOWS-X86-64: "{{.*}}link{{(.exe)?}}"
-- 
2.37.0.rc0.161.g10f37bed90-goog

